sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/AdtQueryBuilder.cs

```diff
@@ -89,27 +91,62 @@ public FromQuery SelectCount()
         /// </example>
         /// <param name="customQuery"> Query in string format. </param>
         /// <returns> Query that contains a select clause. </returns>
-        public FromQuery SelectCustom(string customQuery)
+        public SelectAsQuery SelectCustom(string customQuery)
         {
             return _selectQuery.SelectCustom(customQuery);
         }
 
+        /// <summary>
+        /// Used to alias selectable properties in place of the AS keyword.
+        /// </summary>
+        /// <param name="literal"> The proper name for the selectable property in the ADT Query Language. </param>
```

Comment:
`literal` can be changed to `property` or `field` or something else, I don't think it is the right parameter name.